user/urls.py

```diff
@@ -0,0 +1,11 @@
+from django.urls import path
+from . import views
+
+
+app_name = 'user'
+urlpatterns = [
+    path('', views.signup_view, name="signup"),
+    path('password', views.password_view, name="create_password"),
+    path('confirm', views.signup_confirm_view, name="confirm"),
+    path('thanks', views.signup_thanks_view, name="thanks")
```

Comment:
すみません，こちらもチェック漏れでしたが，pathの第一引数の最後にスラッシュを入れるのが通例なのでつけるようにしてください！
※空文字（''）では不要です。
```suggestion
    path('password/', views.password_view, name="create_password"),
    path('confirm/', views.signup_confirm_view, name="confirm"),
    path('thanks/', views.signup_thanks_view, name="thanks")
```